src/uninitialized_device.rs

```diff
@@ -115,30 +118,76 @@ impl<SpiBus: Bus> UninitializedDevice<SpiBus> {
         RawDevice::new(self.bus)
     }
 
+    /// Get the currently set Retry Time-value Register.
+    ///
+    /// RTR (Retry Time-value Register) [R/W] [0x0019 – 0x001A] [0x07D0]
+    ///
+    /// E.g. 4000 = 400ms
+    #[inline]
+    pub fn current_retry_timeout(&mut self) -> Result<RetryTime, SpiBus::Error> {
```

Comment:
Return it as something like a `fugit::Duration` or an `embedded_time::Duration`. I don't think we should be managing time in this crate directly. Alterantively, simplest approach is to just return an int representing the number of millis.

Reply:
Because then the user of the crate will need to use `fugit::Duration` and the more complex type can actually be bad when using the crate.